sign/ed25519/ed25519_test.go

```diff
@@ -39,6 +39,24 @@ func TestMalleability(t *testing.T) {
 	}
 }
 
+func TestPublic(t *testing.T) {
+	var zero zeroReader
+	pub, priv, err := ed25519.GenerateKey(zero)
+	if !priv.Equal(priv) {
+		t.Fatal()
+	}
+	if !pub.Equal(pub) {
+		t.Fatal()
```

Comment:
Why no message with `Fatal`? If you intend to use no message, just use `FailNow()` directly.

https://golang.org/pkg/testing/#T.Fatal

Reply:
Does it really matter?  (Changed anyway.)

Reply:
It does matter: FailNow doesn't show the line number when triggered and Fatal does.  Thus I reverted back to using Fatal.